src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -37,6 +38,20 @@
 
 logger = logging.getLogger(__name__)
 
+UpdatedBatchJobInfo = namedtuple('UpdatedBatchJobInfo', (
+    'jobID',
+    'exitStatus',  # The exit status (integer value) of the job. 0 implies successful.
```

Comment:
The Mesos batch system puts a 255 here if the exit status is unavailable, whereas it looks like the Kubernetes batch system and single machine batch system use -1. We should probably settle on one (or say that either is allowed) and document that as part of the spec for all the batch systems.

Reply:
Good point! Created a constant (decided to use 255) and used it everywhere.